_posts/molecules/2018-10-01-autocomplete.html

```diff
@@ -3,35 +3,21 @@
 title:  Autocomplete
 category: molecules
 description: Whenever a field has the ability to autocomplete its value, we should add a hint in form of placeholder and 
-             also an icon on the left that shows a spinner while loading from remote
+             also an icon on the left that shows a spinner while loading from remote.
+             OBS provides an autocomplete parti, including javascript for handling the autocomplete request and spinner. For
+             further customization of autocomplete events please refer to the jquery autocomplete documentation (https://jqueryui.com/autocomplete/).
```

Comment:
It should be `jQuery` and the link should be `[jQuery autocomplete documentation](https://jqueryui.com/autocomplete/)`